src/plugins/auth.js

```diff
@@ -36,20 +36,29 @@ export const auth = {
          */
         validate: (artifacts) => {
           const user = artifacts.decoded.payload
-          const groups = Array.isArray(user?.groups) ? user.groups : []
 
-          logger.debug(`Validating user against groups: ${groups.join(', ')}`)
+          if (!user) {
+            return {
+              isValid: false
+            }
+          }
+
+          const groups = Array.isArray(user.groups) ? user.groups : []
+
+          logger.debug(
+            `User ${user.preferred_username}: validating against groups: ${groups.join(', ')}`
```

Comment:
Are we alright with how this line logs empty `[]` groups?

Reply:
Enhancement in PR https://github.com/DEFRA/forms-manager/pull/176.

Commit https://github.com/DEFRA/forms-manager/pull/176/commits/71fd2590aff2f04b02aebd083d3e2ae23c975a93.